book/5-data-mgmt-exp-proc/e2e-workflow.md

```diff
@@ -0,0 +1,155 @@
+# What are the Processes Involved to Build a Basic End-to-End Workflow?
```

Comment:
Would this sound better?
# What are the Processes Involved in Building a Basic End-to-End Workflow?

Reply:
Yup. I've changed the title to the recommended one 👍🏼